docs/api-details.md

```diff
@@ -27,37 +30,38 @@ The available calls within an executor are:
 |[clean()](api-details.md#executorclean) | Async. | Method used to clean the temporary data generated by Lithops|
 
 
-**ibm_cf_executor(\*\*kwargs)**
+**LocalhostExecutor(\*\*kwargs)**
 
-Initialize and return an IBM Cloud Functions executor object.
+Initialize and return Localhost executor object.
 
 |Parameter | Default | Description|
 |---|---|---|
 |config | None | Settings passed in here will override those in lithops_config|
 |runtime |  None | Name of the docker image to run the functions |
-|runtime_memory | 256 | Memory (in MB) to use to run the functions |
-|storage_backend | ibm_cos | Storage backend to store temp data|
+|workers | cpu_count | Max number of parallel workers |
```

Comment:
@JosepSampe workers = cpu_count? Why did you removed runtime_memory?

Reply:
@gilv localhost and standalone backends has no `runtime_memory` param. `runtime_memory `is specific param of serverless backends

Reply:
@gilv localhost and standalone backends has cpu_count as max workers